src/sensor_mpu9250.c

```diff
@@ -182,6 +176,9 @@ mp9250_start(struct mpu9250 *mp, uint8_t oid)
     msg[1] = SET_USER_FIFO_EN; // enable FIFO buffer access
     i2c_write(mp->i2c->i2c_config, sizeof(msg), msg);
 
+    uint8_t int_reg[] = {AR_INT_STATUS}; // clear FIFO overflow flag
+    i2c_read(mp->i2c->i2c_config, sizeof(int_reg), int_reg, 1, msg);
```

Comment:
gcc wont know that `int_reg[]` has limited lifetime and thus will allocate stack space for it.  It's better to just reuse the existing `msg[]` allocation.

Reply:
Done!